examples/training/classification/imagenet/basic_training.py

```diff
@@ -110,6 +128,8 @@
 try:
     tpu = tf.distribute.cluster_resolver.TPUClusterResolver.connect()
     strategy = tf.distribute.TPUStrategy(tpu)
+    if FLAGS.use_mixed_precision:
+        keras.mixed_precision.set_global_policy("mixed_bfloat16")
```

Comment:
In the case of mixed precision training, should the model also be modified to produce float32 output?

From the [docs](https://www.tensorflow.org/guide/mixed_precision#summary):
> (...) regardless of what your model ends in, make sure the output is float32.

Or is this a no big deal?

Reply:
Mixed precision doesn't turn all operations into float16. It keeps float32 for operations that require it for numerical stability, like `softmax` does. So I don't think it would actually make a difference here?

Reply:
Got it, thanks :)